src/cls/IPM/Main.cls

```diff
@@ -2594,22 +2696,43 @@ ZPM(pArgs...)
  If $System.CLS.IsMthd("%IPM.Main", "Shell") { 
  	Do ##class(%IPM.Main).Shell(pArgs...) Quit
  } Else {
-	// TODO: Decide what to do here; definitely needs to support enabling IPM from here as well,
-	// but need to decide what level of customization to provide
+	// TODO: Needs to support enabling IPM from here, also need to decide what level of customization to provide
+	Write !, "IPM is not enabled in this namespace."
```

Comment:
Recall: we need to add better error handling based on pArgs. This is used from two contexts, zpm "command":1:1 (say) and $zpm("command"). In the former, if the flag to terminate on error is set, we should terminate if IPM is not available. In the latter, if IPM is not available we want to return an error status.